assets/sass/patterns/organisms/content-aside.scss

```diff
@@ -13,23 +13,25 @@
   }
 
   @media only screen and (min-width: #{get-rem-from-px($bkpt-site--extended-wide)}em) {
-    @include padding(86, "top");
     align-items: flex-end;
     align-self: flex-start;
     display: flex;
     flex-direction: column;
+    width: 270px;
     z-index: 5;
 
     & > .divider {
       display: none;
     }
 
     &.content-aside__sticky {
```

Comment:
This can be placed into the content-aside class so we do not need to have content-aside-sticky class as this is only going to be available in this breakpoint

Reply:
The class "content-aside__sticky" is removed.